Tests/MongoSwiftTests/LoggingTests.swift

```diff
@@ -0,0 +1,158 @@
+import Foundation
```

Comment:
There are eventually going to be spec tests for this, and our testing needs and approach might change quite a bit. 

This was just enough basic code to consume log messages in the tests and confirm things are working as I would expect.